recipes/freetakserver/recipe.yaml

```diff
@@ -0,0 +1,111 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/73cd2eed94c576213c5f25ab57adf6d8c83e792a/schema.json
+schema_version: 1
+
+context:
+  name: freetakserver
+  version: 2.2.2
+  python_min: 3.11.0
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/babeloff/FreeTakServer/archive/refs/tags/v2.2.2-conda-forge.tar.gz
+  sha256: 47baced5d70ac06f14f283a2b30a65209f0a3e1e0afae9fe591e88496d07b155
+
+build:
+  number: 0
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip:
+    - win
+  python:
+    entry_points:
+      - swagger_server=swagger_server.__main__:main
+
+requirements:
+  host:
+    - python ==${{ python_min }}
+    - poetry-core
+    - pip
+  run:
+    - python
+    - bitarray
+    - click >=8.1.3
+    - colorama 0.4.4.*
```

Comment:
Most versions only have 3 parts e.g. `colorama` in this case I believe the extra `.*` is surperfluous and the effect is to have a hard-pin on the exact patch version of `colorama` being used.

It's almost *always* a very bad idea to have hard pin to patch versions as it means the app will never get even security fixes and it will pretty much guarantee that it won't be  able to be installed alongside anything else.

The usual thing to do is use `>=` for a minimum required version, and if you want, pin to the major version as well.